test/test.mjs

```diff
@@ -52,12 +60,32 @@ assert.equal(
 
 <!-- has a comment -->
 
+<!--
+has a multi-line comment 
+-->
+
+<!-- another 
+multi-line 
+comment -->
+
 and a paragraph
 `,
     { ast: true }
   ),
-  '# This <!-- inline -->document\n\n<!-- has a comment -->\n\nand a paragraph\n'
-)
+    `# This <!-- inline -->document
+
+<!-- has a comment -->
+
+<!--
+has a multi-line comment 
+-->
+
+<!-- another 
+multi-line 
+comment -->
```

Comment:
Apart the refactor to a template literal, only those lines were added and the rest remains unchanged